pkg_client/src/components/PKGVisualization.tsx

```diff
@@ -0,0 +1,83 @@
+// Natural language to PKG component
+
+import { Container } from "react-bootstrap";
+import { UserContext } from "../contexts/UserContext";
+import { useContext, useEffect, useState } from "react";
+import axios from "axios";
+import QueryForm from "./QueryForm";
+import Button from "react-bootstrap/Button";
+
+const PKGVisualization = () => {
+    const { user } = useContext(UserContext);
+    const [error, setError] = useState("");
+    const [image_path, setImagePath] = useState("");
```

Comment:
Nit: usually in javascript, the variable name uses a capital letter instead of `_`, e.g., `imagePath`.